src/test/resources/test.properties

```diff
@@ -0,0 +1,7 @@
+baseUrl = http://automatizacion.herokuapp.com/azapata/
+pathToChromeDriver = chromedriver.exe
+completeName = Andres
+lastName = Zapata
+phone = 3124578900
+identification_type = Cédula de ciudadanía
```

Comment:
not sure if this will work since have special characters and withespaces. It is not required to do any change right now, but if something is failing when you try to read this property you can use "Cédula de ciudadanía"